cmake/llvm-toolchain.cmake

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+include_guard(GLOBAL)
+
+set(CMAKE_C_COMPILER clang)
+set(CMAKE_CXX_COMPILER clang++)
+
+if(BEMAN_BUILDSYS_SANITIZER STREQUAL "ASan")
```

Comment:
ASan already has a well-known meaning in the community: address sanitizer. Using the same word to mean "all sanitizers" is confusing.

What about "MaxSan" (maximum sanitizers) which would mean getting as many sanitizers enabled as possible.

Reply:
👌 good suggestion

Reply:
This is my fault, to the extent this is coming from optional26. Address composes with many others, and I didn't try to find a better name for the collection. MaxSan is at least not wrong, and I don't have a better idea.

Reply:
Adopted!